README.md

```diff
@@ -127,6 +127,45 @@ $ pytest -v
 
 Done! You are ready to develop for the RMM OSS project.
 
+### Caching third-party dependencies
+
+RMM uses [CPM.cmake](https://github.com/TheLartians/CPM.cmake) to
+handle third-party dependencies like spdlog, Thrust, GoogleTest,
+GoogleBenchmark. In general you won't have to worry about it. If cmake
+finds an appropriate version on your system, it'll use that (you can
+help that along by setting `CMAKE_PREFIX_PATH` to point to the
+installed location. Otherwise those dependencies will be downloaded as
+part of the build.
+
+If you frequently start new builds from scratch, you might want to
+consider setting the environment variable `CPM_SOURCE_CACHE` to an
+external download directory, which will avoid repeated downloads of
+the thirdparty dependencies.
+
+## Using RMM in a downstream CMake project
+
+If you want to use RMM in your own CMake project, the installed RMM
+library will provide a set of config files that makes it easy to
+integrate it. In your `CMakeLists.txt`, just add
+
+```cmake
+find_package(rmm [VERSION])
+# ...
+target_link_libraries(<your-target> (PRIVATE|PUBLIC) rmm::rmm)
+```
+
+Since RMM is a header-only library, this will not actually link RMM,
+but make the headers available, as well as pull in transitive
+dependencies. If RMM is not installed in a default location, use
```

Comment:
```suggestion
Since RMM is a header-only library, this does not actually link RMM,
but it makes the headers available and pulls in transitive dependencies.
If RMM is not installed in a default location, use
```